src/app/e2e/home_published.js

```diff
@@ -51,6 +53,69 @@ describe('Home page with published data', function homePublishedDataTests() {
         expect(tdsText.some(t => t === 'peregrin.took@shire.net')).toEqual(true);
     });
 
+    it('should go to detail page when clicking on uri', async () => {
+        const firstUriLink = await driver.findElement(By.linkText('1'));
+        const firstUri = await firstUriLink.getText();
+        firstUriLink.click();
+        await driver.wait(until.elementLocated(By.css('.title')));
+        const title = await driver.findElement(By.css('.title, h1'), DEFAULT_WAIT_TIMEOUT);
+        driver.wait(until.elementTextIs(title, firstUri), DEFAULT_WAIT_TIMEOUT);
+    });
+
+    it('should display all resource properties', async () => {
+        await driver.wait(until.elementLocated(By.css('.detail')), DEFAULT_WAIT_TIMEOUT);
+        const fullnameLabel = await driver.findElement(By.css('.detail .property:nth-child(2) dt'));
+        expect(await fullnameLabel.getText()).toEqual('fullname\nhttp://www.w3.org/ns/person');
+
+        const fullnameValue = await driver.findElement(By.css('.detail .property:nth-child(2) dd'));
+        expect(await fullnameValue.getText()).toEqual('PEREGRIN.TOOK');
+
+        const mailLabel = await driver.findElement(By.css('.detail .property:last-child dt'));
+        expect(await mailLabel.getText()).toEqual('email\nhttp://uri4uri.net/vocab');
+
+        const mailValue = await driver.findElement(By.css('.detail .property:last-child dd'));
+        expect(await mailValue.getText()).toEqual('peregrin.took@shire.net');
+    });
+
+    it('should allow to add field resource properties', async () => {
+        await driver.findElement(By.css('.add-field-resource')).click();
+        await driver.wait(until.elementLocated(By.css('.detail-properties')), DEFAULT_WAIT_TIMEOUT);
+        const form = driver.findElement(By.css('#add_field_resource_form'));
+
+        const contributorName = form.findElement(By.css('.contributor-name input'));
+        contributorName.sendKeys('john');
```

Comment:
same for all other inputs. We might want to introduce `untilElementHasBeenSentKeys`